streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfo.java

```diff
@@ -120,8 +122,17 @@ public int latestSupportedVersion() {
         return standbyTasks;
     }
 
-    public Map<HostInfo, Set<TopicPartition>> partitionsByHost() {
-        return partitionsByHost;
+    public Map<HostInfo, Set<TaskId>> tasksByHost() {
+        return tasksByHost;
+    }
+
+    private byte[] toBytes(final int i) {
```

Comment:
Might be better to use `ByterBuffer.putInt(...).array()` instead of this method?